pytests/test_sparse_transforms.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_almost_equal
+
+from pylops.signalprocessing import DCT
+from pylops.utils import dottest
+
+par1 = {"ny": 11, "nx": 11, "imag": 0, "dtype": "float64"}
+par2 = {"ny": 11, "nx": 21, "imag": 0, "dtype": "float64"}
+par3 = {"ny": 21, "nx": 21, "imag": 0, "dtype": "float64"}
+
+
+@pytest.mark.parametrize("par", [(par1), (par3)])
+def test_DCT1D(par):
+    """Dot test for Discrete Cosine Transform Operator 1D"""
+
+    t = np.arange(par["ny"])
+    # testing for various types of dct
+    Dct = DCT(dims=(par["ny"],), type=1, dtype=par["dtype"])
```

Comment:
Please use a for loop

```
for type in [1, 2, 3, 4]:
...
```

Reply:
added loops for repetitive code in all test cases [dcf810a](https://github.com/PyLops/pylops/pull/480/commits/dcf810a9da5c1f0f07ba72d09d5cf34a32fc0c88)